diff -rupN xf86-video-amdgpu-master/src/amdgpu_kms.c xf86-video-amdgpu-git/src/amdgpu_kms.c
--- xf86-video-amdgpu-master/src/amdgpu_kms.c	2020-05-27 17:41:22.000000000 +0200
+++ xf86-video-amdgpu-git/src/amdgpu_kms.c	2020-05-31 16:13:13.261356600 +0200
@@ -1682,6 +1682,8 @@ Bool AMDGPUPreInit_KMS(ScrnInfoPtr pScrn
 	} else {
 		info->cursor_w = CURSOR_WIDTH_CIK;
 		info->cursor_h = CURSOR_HEIGHT_CIK;
+		info->cursor_w = CURSOR_WIDTH;
++		info->cursor_h = CURSOR_HEIGHT;
 	}
 
 	amdgpu_query_heap_size(pAMDGPUEnt->pDev, AMDGPU_GEM_DOMAIN_GTT,
