diff -Naur xf86-video-amdgpu-git/src/amdgpu_kms.c patched-xf86-video-amdgpu-git/src/amdgpu_kms.c
--- xf86-video-amdgpu-git/src/amdgpu_kms.c	2020-07-18 12:38:08.000000000 +0200
+++ patched-xf86-video-amdgpu-git/src/amdgpu_kms.c	2020-07-18 13:29:29.638771800 +0200
@@ -1682,6 +1682,8 @@
 	} else {
 		info->cursor_w = CURSOR_WIDTH_CIK;
 		info->cursor_h = CURSOR_HEIGHT_CIK;
+		info->cursor_w = CURSOR_WIDTH;
+		info->cursor_h = CURSOR_HEIGHT;
 	}
 
 	amdgpu_query_heap_size(pAMDGPUEnt->pDev, AMDGPU_GEM_DOMAIN_GTT,
