diff -Nur /libdrm-git/src/libdrm/data/amdgpu.ids /libdrm-git-patch/src/libdrm/data/amdgpu.ids
--- /libdrm-git/src/libdrm/data/amdgpu.ids	2021-10-24 19:20:46.372398000 +0200
+++ /libdrm-git-patch/src/libdrm/data/amdgpu.ids	2021-11-11 20:14:46.353519000 +0100
@@ -298,3 +298,6 @@
 9874,	84,	AMD Radeon R7 Graphics
 6FDF,	E7,	AMD Radeon RX 590 GME
 6FDF,	EF,	AMD Radeon RX 580 2048SP
+9920,	0,	AMD Liverpool (PS4)
+9922,	0,	AMD Liverpool (PS4)
+9923,	0,	AMD Liverpool (PS4)
diff -Nur /libdrm-git/src/libdrm/radeon/r600_pci_ids.h /libdrm-git-patch/src/libdrm/radeon/r600_pci_ids.h
--- /libdrm-git/src/libdrm/radeon/r600_pci_ids.h	2021-10-24 19:20:46.382398000 +0200
+++ /libdrm-git-patch/src/libdrm/radeon/r600_pci_ids.h	2021-11-11 20:13:09.995608000 +0100
@@ -416,6 +416,10 @@
 CHIPSET(0x665D, BONAIRE_665D, BONAIRE)
 CHIPSET(0x665F, BONAIRE_665F, BONAIRE)
 
+CHIPSET(0x9920, LIVERPOOL_9920, LIVERPOOL)
+CHIPSET(0x9922, LIVERPOOL_9922, LIVERPOOL)
+CHIPSET(0x9923, LIVERPOOL_9923, LIVERPOOL)
+
 CHIPSET(0x9830, KABINI_9830, KABINI)
 CHIPSET(0x9831, KABINI_9831, KABINI)
 CHIPSET(0x9832, KABINI_9832, KABINI)
diff -Nur /libdrm-git/src/libdrm/radeon/radeon_surface.c /libdrm-git-patch/src/libdrm/radeon/radeon_surface.c
--- /libdrm-git/src/libdrm/radeon/radeon_surface.c	2021-10-24 19:20:46.382398000 +0200
+++ /libdrm-git-patch/src/libdrm/radeon/radeon_surface.c	2021-11-11 20:14:02.542958000 +0100
@@ -85,6 +85,7 @@
     CHIP_OLAND,
     CHIP_HAINAN,
     CHIP_BONAIRE,
+    CHIP_LIVERPOOL,
     CHIP_KAVERI,
     CHIP_KABINI,
     CHIP_HAWAII,
