diff -rupN drm-master/data/amdgpu.ids libdrm-git/data/amdgpu.ids
--- drm-master/data/amdgpu.ids	2020-05-27 23:16:28.000000000 +0200
+++ libdrm-git/data/amdgpu.ids	2020-06-01 09:56:14.506767800 +0200
@@ -221,3 +221,7 @@
 9874,	84,	AMD Radeon R7 Graphics
 6FDF,	E7,	AMD Radeon RX 590 GME
 6FDF,	EF,	AMD Radeon RX 580 2048SP
+9920,	00, AMD Liverpool (PlayStation 4)
+9922,	00, AMD Liverpool (PlayStation 4)
+9923,	00, AMD Liverpool (PlayStation 4)
+9924,	11, AMD Gladius (PlayStation 4)
diff -rupN drm-master/radeon/r600_pci_ids.h libdrm-git/radeon/r600_pci_ids.h
--- drm-master/radeon/r600_pci_ids.h	2020-05-27 23:16:28.000000000 +0200
+++ libdrm-git/radeon/r600_pci_ids.h	2020-06-01 09:56:58.537392700 +0200
@@ -473,6 +473,11 @@ CHIPSET(0x131B, KAVERI_131B, KAVERI)
 CHIPSET(0x131C, KAVERI_131C, KAVERI)
 CHIPSET(0x131D, KAVERI_131D, KAVERI)
 
+CHIPSET(0x9920, LIVERPOOL_9920, LIVERPOOL)
+CHIPSET(0x9922, LIVERPOOL_9922, LIVERPOOL)
+CHIPSET(0x9923, LIVERPOOL_9923, LIVERPOOL)
+CHIPSET(0x9924, GLADIUS_9924, GLADIUS)
+
 CHIPSET(0x67A0, HAWAII_67A0, HAWAII)
 CHIPSET(0x67A1, HAWAII_67A1, HAWAII)
 CHIPSET(0x67A2, HAWAII_67A2, HAWAII)
diff -rupN drm-master/radeon/radeon_surface.c libdrm-git/radeon/radeon_surface.c
--- drm-master/radeon/radeon_surface.c	2020-05-27 23:16:28.000000000 +0200
+++ libdrm-git/radeon/radeon_surface.c	2020-06-01 09:57:30.996933600 +0200
@@ -86,6 +86,8 @@ enum radeon_family {
     CHIP_HAINAN,
     CHIP_BONAIRE,
     CHIP_KAVERI,
+	CHIP_LIVERPOOL,
+    CHIP_GLADIUS,
     CHIP_KABINI,
     CHIP_HAWAII,
     CHIP_MULLINS,
